src/pretalx/orga/templates/orga/schedule/room_form.html

```diff
@@ -15,6 +15,7 @@
         {% bootstrap_form_errors form %}
         <h2>{% translate "Room" context "Room information" %}</h2>
         {% bootstrap_field form.name layout='event' %}
+        {% bootstrap_field form.guid layout='event' %}
```

Comment:
I think it would be best to hide/collapse this setting, because it's only relevant in very restricted cases and will absolutely confuse people otherwise. An expandable "advanced settings" section sounds pretty good for this.

Reply:
I will move the field to the bottom, but do not know how to make it collapsible.